windows-msi/msi.wxs

```diff
@@ -387,10 +568,68 @@
                         <File Name="license.txt" Source="!(bindpath.build)$(var.PRODUCT_PLATFORM)\license.txt"/>
 
                         <!-- Clean-up NSIS installation. -->
```

Comment:
This is not the right place to comment on this but finding the right commit in git PRs is hard when the patch big and commits numerous.. Or I don't know how to tame it on a browser -- much easier with git and vi :)

Anyway, the registry entry disable_save_passwords is DWORD but here it is read back as raw -- that will set it to "#0" or "#1" and when written back as integer it will become REG_SZ (WiX magic, I guess).

An option is to set its default to "#0", read as raw, and then write as string which will automatically become DWORD because of the # prefix  --- I suppose. Or you may know a better way.

Reply:
You're right. Missed that. I checked the final MSI and the WiX indeed authored the `Value` column of `Registry` table to `#[DISABLESAVEPWD]`, which will render incorrectly to `##0` or `##1` on updates.

I see no better to address this other than you suggested.